src/cache/cache_las.c

```diff
@@ -572,6 +583,80 @@ __las_squash_modifies(WT_SESSION_IMPL *session, WT_CURSOR *cursor, WT_UPDATE **u
     return (ret);
 }
 
+/*
+ * __las_insert_record --
+ *     A helper function to insert the record into the lookaside including stop time pair.
+ */
+static int
+__las_insert_record(WT_SESSION_IMPL *session, WT_CURSOR *cursor, uint64_t btree_id, WT_ITEM *key,
+  WT_UPDATE *upd, wt_timestamp_t *stop_ts, uint64_t *stop_txnid)
+{
+    WT_ITEM las_value;
+
+    WT_CLEAR(las_value);
+
+    /* Check the update type and get the value */
+    switch (upd->type) {
+    case WT_UPDATE_MODIFY:
+    case WT_UPDATE_STANDARD:
+        las_value.data = upd->data;
+        las_value.size = upd->size;
+        break;
+    case WT_UPDATE_TOMBSTONE:
+        las_value.size = 0;
+        break;
+    default:
+        /*
+         * It is never OK to see a birthmark here - it would be referring to the wrong page image.
+         */
+        WT_RET(__wt_illegal_value(session, upd->type));
+    }
+
+    cursor->set_key(cursor, btree_id, key, upd->start_ts, upd->txnid, *stop_ts, *stop_txnid);
+
+    /*
+     * Set the current update start time pair as the commit time pair to the lookaside record.
```

Comment:
It's usual to keep the comment on a single line if possible - and I think you can here.

Reply:
corrected.